R/appender.R

```diff
@@ -97,3 +109,29 @@ appender.tee <- function(file){
     cat(line, file=file, append=TRUE, sep='')
   }
 }
+
+# Write to a dynamically-named file (and optionally the console)
+appender.file2 <- function(format, console = FALSE,
+                           default.func = "shell", datetime.fmt = "%Y%m%dT%H%M%S"){
```

Comment:
Some consistency items:
+ please no spaces between `=` in named argument assignment;
+ space between `)` and `{`;
+ use two spaces for function line wrapping (otherwise situation degenerates quickly).